taca/storage/storage.py

```diff
@@ -169,29 +170,36 @@ def cleanup_uppmax(site, days, dry_run=False):
     deleted_log = CONFIG.get('cleanup').get('deleted_log')
     assert os.path.exists(os.path.join(root_dir,deleted_log)), "Log directory {} doesn't exist in {}".format(deleted_log,root_dir)
     log_file = os.path.join(root_dir,"{fl}/{fl}.log".format(fl=deleted_log))
+    list_to_delete = []
 
     # make a connection for project db #
     pcon = statusdb.ProjectSummaryConnection()
     assert pcon, "Could not connect to project database in StatusDB"
 
-    if site != "archive":
+    if site in ["analysis", "illumina"]:
         ## work flow for cleaning up illumina/analysis ##
         projects = [ p for p in os.listdir(root_dir) if re.match(filesystem.PROJECT_RE,p) ]
-        list_to_delete = get_closed_projects(projects, pcon, days)
-    else:
+        list_to_delete.extend(get_closed_projects(projects, pcon, days))
+    elif site == "archive":
         ##work flow for cleaning archive ##
-        list_to_delete = []
-        archived_in_swestore = filesystem.list_runs_in_swestore(path=CONFIG.get('cleanup').get('swestore').get('root'), no_ext=True)
+        archive_config = CONFIG.get('cleanup').get('archive')
         runs = [ r for r in os.listdir(root_dir) if re.match(filesystem.RUN_RE,r) ]
-        with filesystem.chdir(root_dir):
-            for run in runs:
-                fc_date = run.split('_')[0]
-                if misc.days_old(fc_date) > days:
-                    if run in archived_in_swestore:
-                        list_to_delete.append(run)
-                    else:
-                        logger.warn("Run {} is older than {} days but not in "
-                                    "swestore, so SKIPPING".format(run, days))
+        for run in runs:
+            with filesystem.chdir(os.path.join(root_dir, run)):
+                ## Collect all project path in the run folder
+                all_proj_path = glob(archive_config.get('proj_path'))
```

Comment:
Yep I meant the second with an example... no problem if the example will be no longer valid after three months, he give an idea to who is looking at the code about what is happening without the need of going into the config file